src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -49,6 +60,57 @@ internal bool AutomaticRedirection
                 _automaticRedirection = value;
             }
         }
+        internal bool UseProxy
+        {
+            get
+            {
+                if (_proxyPolicy == CurlProxyUsePolicy.UseDefaultProxy)
+                {
+                    return false;
+                }
+                return _proxyPolicy != CurlProxyUsePolicy.DoNotUseProxy;
```

Comment:
It is not logically equivalent.  If the value is CurlProxyUsePolicy.UseDefaultProxy, then this will return FALSE.  But that is not correct.  The HttpClientHandler.UseProxy Boolean should be true whether it is a 'default' proxy or a 'custom' proxy which uses the Proxy property IWebProxy.  The only time that HttpClientHandler.UseProxy should be FALSE if if the policy is .DoNotUsePolicy.